jkube-kit/common/src/test/java/org/eclipse/jkube/kit/common/util/SpringBootUtilTest.java

```diff
@@ -33,10 +36,14 @@
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertNull;
 import static org.junit.Assert.assertTrue;
-
+import static org.mockito.Mockito.when;
+@RunWith(MockitoJUnitRunner.class)
 public class SpringBootUtilTest {
     @Rule
     public TemporaryFolder temporaryFolder = new TemporaryFolder();
+    @Mock
+    JavaProject maven_project;
```

Comment:
Could you please use Java naming conventions for the variable declaration? (I know the original variable was like this, but we definitely need to change this)
```suggestion
    JavaProject mavenProject;
```